app/indexers/hyrax/valkyrie_collection_indexer.rb

```diff
@@ -7,6 +7,7 @@ class ValkyrieCollectionIndexer < Hyrax::ValkyrieIndexer
     include Hyrax::ResourceIndexer
     include Hyrax::PermissionIndexer
     include Hyrax::VisibilityIndexer
+    include Hyrax::Indexer(:core_metadata)
```

Comment:
Collection resource defines core metadata, so seems like the collection indexer should index it.

Reply:
:+1: